source/ControllableHero.hx

```diff
@@ -0,0 +1,33 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+
+class ControllableHero extends FlxSprite
+{
+	public function new(x:Float = 0, y:Float = 0)
+	{
+		super(x, y);
+
+		makeGraphic(20, 20, 0xFFD4AF37);
```

Comment:
Numbers like these `20`s can be pretty mysterious, especially if we step away from the codebase for  a while. 

Are these both w0 because they were supposed to always match one another? Were they independently picked, and can be changed willy-nilly without issues? Are there any other spots in the code that depend on these staying unchanged?

It's good practice to take numbers like these and store them in class-level variables - static properties that can't be set to something new later is usually a good policy for const-type stuff.

Then we can give them names and make the intent of our code clearer - the player's starting_width should be 15, the player's starting_height should be 15.

Let's implement this sort of fix here and elsewhere in the project.